newIDE/app/src/ObjectEditor/Editors/Cube3DEditor.js

```diff
@@ -196,6 +200,35 @@ const Cube3DEditor = ({
             />
           ))}
         </SelectField>
+        <SelectField
+          value={properties.get('materialType').getValue()}
+          floatingLabelText={properties.get('materialType').getLabel()}
+          helperMarkdownText={properties.get('materialType').getDescription()}
+          onChange={(event, index, newValue) => {
+            onChangeProperty('materialType', newValue);
+          }}
+        >
+          <SelectOption
+            label={t`Always lighted (no light needed)`}
+            value="AlwaysLighted"
+            key="AlwaysLighted"
+          />
+          <SelectOption
+            label={t`Emit all ambient light`}
+            value="EmitAllAmbientLight"
+            key="EmitAllAmbientLight"
+          />
+        </SelectField>
+        {properties.get('materialType').getValue() !== 'AlwaysLighted' &&
+          !hasLight(layout) && (
+            <AlertMessage kind="error">
+              <Trans>
+                For standard materials reacting to lights, make sure to set up a
```

Comment:
Same as for model:

We need to use the same terminology as in the dropdown. So if we use "standard materials" here, we should use standard material.
With my suggestion, we can use "For realistic material (also called standard material), reacting to lights, make sure [...]"